pkg/controller/volume/persistentvolume/framework_test.go

```diff
@@ -384,10 +392,12 @@ func checkEvents(t *testing.T, expectedEvents []string, ctrl *PersistentVolumeCo
 			} else {
 				klog.V(5).Infof("event recorder finished")
 				finished = true
+				break
```

Comment:
timer wasn't working without this

Reply:
can you clarify what you mean by that? I don't understand why this was required

Reply:
reverted this is not needed, I borked the test while debugging and blamed the timer